.gitignore

```diff
@@ -1,4 +1,8 @@
 .DS_Store
+cypress/fixtures
+cypress/plugins
+cypress/support
+cypress/videos
```

Comment:
These folders are auto-generated by cypress when running `cypress open` command. These are not modified so I have ignored these files on check-in.